packages/hash/frontend/src/components/layout/PageHeader/PageHeader.tsx

```diff
@@ -33,27 +31,11 @@ export const PageHeader: React.VFC<{
   accountId: string;
 }> = ({ accountId }) => {
   const theme = useTheme();
```

Comment:
In the PR description it says:
> [Destructure the theme object](https://github.com/hashintel/hash/pull/397/files#diff-795901c71d9a5e5302bdb94610f90971b7b5e2f971fcde694d1d688338b3b561R60) instead of using useTheme.

Could we also address this here please?

Reply:
Will do. It seems the only intended usecase of `useTheme` is when used within the React render loop but outside the JSX?

Reply:
Done here: https://github.com/hashintel/hash/pull/397/commits/5f616c622330c7ecdd90a63c95e8ca3ced87f11c

Reply:
We can use it for all cases @akash-joshi. We are just sticking to just using it in the scenario you pointed out for consistency

Reply:
> It seems the only intended usecase of useTheme is when used within the function body but outside the JSX?

@akash-joshi I think the intended use-case is actually just more flexibility, but yes that's how I think we should consistently use it. When you're accessing the them in the `sx` prop, always use the callback's arguments. When you need to access the theme outside of the `sx` prop, that's when `useTheme` should be used.